chim_launcher.py

```
@@ -16,119 +16,78 @@
-                # Log failure only if connection was previously reported as ok AND server was ready
-                if launcher.wsl_connection_reported and launcher.wsl_server_ready:
-                     launcher.after(0, launcher.append_output, f"WSL Connection to Skyrim Lost!\n", "red")
-                # launcher.wsl_connection_reported = False # REMOVE: Don't reset on error
-                self.send_error(400, f"Error reading request body: {e}")
-                # launcher.append_output(f"Proxy Error: Failed reading request body: {e}\n") # Hidden log
-                # update_ui_callback("error", None) # Removed UI update
-                return
-        
+                client_sock, addr = s.accept()
+                threading.Thread(target=self.handle_client, args=(client_sock,), daemon=True).start()
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)